internal/container/plugins.go

```diff
@@ -41,8 +40,6 @@ func Plugins() []interface{} {
 		// service plugin
 		&service.Plugin{},
 
-		// static
-		&static.Plugin{},
```

Comment:
One more question - what is the reason for `static` plugin removal?

Reply:
It's not actually removed; it's reworked. Because previously, it was used as middleware on the main HTTP path. That was very hard to implement a proper static handler, `etags,` and central file server. It was merged into the HTTP plugin with a separate file server on the own path (which can be set by the new config options). As for the config - it stays like previous to have backward compatibility with a few new options.
```yaml
    static:
        dir: "."
        pattern: "/tests/"
        forbid: [""]
        allow: [".txt", ".php"]
        request:
            "input": "custom-header"
        response:
            "output": "output-header"
```

Reply:
see: https://github.com/spiral/roadrunner/pull/646